concepts/project-roles.mdx

```diff
@@ -22,10 +22,6 @@ Once a project is created, you can invite others to collaborate:
 
 Contact us to create more fine-grained roles.
 
-### Managing API Keys
```

Comment:
I'd keep this section - it is unrelated to the shared credentials for LLMs. It'd be great to rename the section header to "Managing Literal API Keys" to avoid confusing with LLM API keys.

Reply:
Done